autoload/airline/extensions/wordcount.vim

```diff
@@ -82,13 +82,15 @@ endfunction
 
 " airline functions {{{1
 " default filetypes:
-let s:filetypes = ['help', 'markdown', 'rst', 'org', 'text', 'asciidoc', 'tex', 'mail']
+let g:airline#extensions#wordcount#filetypes =
+    \ ['help', 'markdown', 'rst', 'org', 'text', 'asciidoc', 'tex', 'mail']
+let s:lastknownfiletypes = g:airline#extensions#wordcount#filetypes
 function! airline#extensions#wordcount#apply(...)
-  let filetypes = get(g:, 'airline#extensions#wordcount#filetypes', s:filetypes)
+  let filetypes = g:airline#extensions#wordcount#filetypes
```

Comment:
why not something like `let filetypes = uniq(sort(s:filetypes + get(g:, 'airline#extensions#wordcount#filetype', []))`

Reply:
Because the global variable is now guaranteed to exist (well, not _guaranteed_, but if it doesn't we have a bigger problem), and `s:filetypes` is no longer a thing. This format allows both adding to and removing from the default set of filetypes without having to replicate it (and thus  desync yourself from whatever vim-airline might do in the future).